Shoko.Server/Scheduling/Jobs/DeleteImportFolderJob.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Threading.Tasks;
+using Quartz;
+
+namespace Shoko.Server.Scheduling.Jobs;
+
+public class DeleteImportFolderJob : IJob
+{
+
+    public static readonly JobKey Key = new("DeleteImportFolder", "System");
```

Comment:
I would change System to Legacy. That's just me, though.
When the queue is moved over to Quartz, I expect I'll separate them by purpose and provider. For example, Import, AniDB, Images, etc

Reply:
The key itself depends on how it handles distinction with job data. The queue is distinct by inputs.